### DISCUSSION TITLE
HW 3 Part 1 Q1

### DISCUSSION THREAD

**Discussion by ewk19001** [cooperrc/computational-mechanics#201] [Q&A]
Hi, I am working on question 1 for homework 3, and my values seem pretty far off. Am I on the right track?

![image](https://github.com/cooperrc/computational-mechanics/assets/135074973/8aae18be-c087-4492-b6b8-8f064812114e)

**Comment by ewk19001** [comment]
I think my data from the notes module was just off, my numbers appear to be better now.

**Reply by cooperrc** [reply]
That's usually the issue: many times the time, e.g. `t` variable is off or the conversion from pixels-to-m is off.
